js/data.js

```diff
@@ -0,0 +1,125 @@
+    technologies: [
+      'css', 'html', 'bootstrap', 'ruby',
+    ],
+  },
+  {
+    id: 'project2',
+    title: 'Multi-Post Strories',
+    screenshot: './images/modal-image.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    technologies: [
+      'css', 'html', 'bootstrap', 'ruby',
+    ],
+  },
+  {
+    id: 'project3',
+    title: 'Multi-Post Strories',
+    screenshot: './images/modal-image.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    technologies: [
+      'css', 'html', 'bootstrap', 'ruby',
+    ],
+  },
+  {
+    id: 'project4',
+    title: 'Multi-Post Strories',
+    screenshot: './images/modal-image.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    technologies: [
+      'css', 'html', 'bootstrap', 'ruby',
+    ],
+  },
+
+];
+
+// Cards being loaded dynamically
+const cardsContainer = document.querySelector('.cards');
+
+for (let i = 0; i < projects.length; i += 1) {
+  const cardElement = document.createElement('div');
+
+  cardsContainer.appendChild(cardElement);
+
+  cardElement.className = 'card';
+
+  const imgPlaceHolder = document.createElement('div');
+
+  cardElement.appendChild(imgPlaceHolder);
+
+  imgPlaceHolder.className = 'img-placeholder';
+
+  const imgElement = document.createElement('img');
+  imgElement.src = './images/img-placeholder.png';
+  imgElement.alt = 'multi-post-stories-image';
+
+  imgPlaceHolder.appendChild(imgElement);
+
+  const descSection = document.createElement('section');
+  descSection.className = 'card-desc';
+
+  cardElement.appendChild(descSection);
+
+  const h3 = document.createElement('h3');
+  h3.innerText = projects[i].title;
+
+  const p = document.createElement('p');
+  p.innerText = projects[i].description;
+
+  descSection.appendChild(h3);
+  descSection.appendChild(p);
+
+  const ul = document.createElement('ul');
+
+  ul.className = 'lines-links';
+  for (let link = 0; link < projects[i].technologies.length; link += 1) {
+    const li = document.createElement('li');
+    li.innerText = projects[i].technologies[link];
+    ul.appendChild(li);
+  }
+
+  cardElement.appendChild(ul);
+
+  const a = document.createElement('a');
+  a.classList.add('btn', 'btn-primary', 'open-modal');
+  a.innerText = 'See Project';
+  a.id = projects[i].id;
+
+  cardElement.appendChild(a);
+}
```

Comment:
- [ ] As it a requirement for this project and a required skill for a front-end developer, that is sticking to the design as much as possible, your `See Project` buttons is overlapping in the main page. See screenshots below and kindly fix that.
***The design***
![yourImplementation](https://user-images.githubusercontent.com/62849872/151149972-84a79b94-65cb-42a8-9ca1-28c01b4432da.png)

***Your implementation***
![seeProject](https://user-images.githubusercontent.com/62849872/151149957-24498e38-dafa-4b3d-94e9-f35c756104cf.png)

Reply:
I do not understand why it is loading like so in your browser but it is loading fine in mine and in my partner's browser. Can we meet on slack?

Reply:
![image](https://user-images.githubusercontent.com/22321030/151159270-26ee67ec-8720-4721-96c3-0c4c936cc840.png)

See screenshot above.

Reply:
I currently have no live project other than this portfolio, come another project I will definitely add the link to the projects array. 
I thought the projects are just placeholders for the future projects that I will have, are they not?

Reply:
Hello @mnekx, the links are just demo links, and should be included in the objects in the projects array. See example below, check to two last lines you can add any demo links. 
![project](https://user-images.githubusercontent.com/62849872/151190053-4a250657-59c3-42c1-a63d-223b1668d4c1.png)